packages/contracts/hardhat.config.ts

```diff
@@ -28,6 +39,45 @@ for (const network of Object.keys(networks)) {
   networks[network].accounts = accounts;
 }
 
+task('build-contracts').setAction(async (args, hre) => {
+  await hre.run('compile');
+  if (
+    hre.network.name === 'zkTestnet' ||
+    hre.network.name === 'zkLocalTestnet'
+  ) {
+    // Copying is useful because we won't have to
+    // change imports in there for artifacts.
+    fs.cpSync('./build/artifacts-zk', './artifacts', {
+      recursive: true,
+      force: true,
+    });
+    fs.cpSync('./build/cache-zk', './cache', {recursive: true, force: true});
+
+    return;
+  }
+
+  fs.cpSync('./build/artifacts', './artifacts', {recursive: true, force: true});
+  fs.cpSync('./build/cache', './cache', {recursive: true, force: true});
+});
+
+task('deploy-contracts').setAction(async (args, hre) => {
+  await hre.run('build-contracts');
+  await hre.run('deploy');
+});
+
+task('test-contracts').setAction(async (args, hre) => {
+  await hre.run('build-contracts');
+  const imp = await import('./test/test-utils/wrapper');
+
+  const wrapper = await imp.Wrapper.create(
+    hre.network.name,
+    hre.ethers.provider
+  );
+  hre.wrapper = wrapper;
+
+  await hre.run('test');
+});
```

Comment:
Not fully sure why we need to overwrite here, perhaps some comments would be helpful